src/server/main_service.cc

```diff
@@ -967,6 +967,7 @@ void Service::Init(util::AcceptServer* acceptor, std::vector<facade::Listener*>
   config_registry.RegisterMutable("max_segment_to_consider");
   config_registry.RegisterMutable("pipeline_squash");
   config_registry.RegisterMutable("lua_mem_gc_threshold");
+  config_registry.RegisterMutable("background_debug_jobs");
```

Comment:
can you please add same support for "dfly expire" ? maybe rename  `background_debug_jobs` to `background_jobs` ?

Reply:
we should make it non-transactional.